plugins/justify/plugin.js

```diff
@@ -178,8 +188,17 @@
 
 		refresh: function( editor, path ) {
 			var firstBlock = path.block || path.blockLimit;
-
-			this.setState( firstBlock.getName() != 'body' && getAlignment( firstBlock, this.editor.config.useComputedState ) == this.value ? CKEDITOR.TRISTATE_ON : CKEDITOR.TRISTATE_OFF );
+			var name = firstBlock.getName();
+
+			// Check if we can give to element proper class, style or current path is body (#455).
+			// IE8, after focus and refresh return path with 'body'
+			if ( ( this.cssClassName && editor.activeFilter.check( name + '(' + this.cssClassName + ')' ) ) ||
+				editor.activeFilter.check( name + '{text-align}' ) ||
+				name == editor.editable().getName() ) {
```

Comment:
Almost good… but what about such case:

```html
<div>Te^xt</div>
```

in div-based editor with following ACF: `div; p{text-align}`? In such case buttons won't be disabled, yet they won't do anything.

Therefore the safest approach would be simply check if `firstBlock` is not `editor.editable()`.